src/components/news-form/news-form.js

```diff
@@ -158,6 +159,18 @@ const NewsForm = ({ article, id }) => {
           </TabPanel>
         ))}
       </form>
+
+      <div className={classes.controlsBlock}>
+        <Link to={config.routes.pathToNews}>
+          <StandardButton
+              id='back'
+              title='Назад'
+              variant='outlined'
+              onClickHandler={() => {}}
+              data-cy='back-btn'
```

Comment:
Why this handler is empty?
You can do smth like this
        ```
  <StandardButton
            title={config.buttonTitles.BACK}
            onClickHandler={() => dispatch(push(pathToNews))}
          />
```

Reply:
done